templates/adduser.html

```diff
@@ -1,43 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
-    <meta charset="UTF-8">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="/static/formstyle.css"> <!-- Link to CSS -->
+    <title>Add New Admin/User</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Reset Password</title>
-    <link rel="stylesheet" href="/static/style.css"> <!-- Link to CSS -->
 </head>
 
 <body>
     <div class="centertextleft">
-        <h1 class="h1smaller">Reset Your Password</h1>
-        <p>Please enter your new password below.</p>
+        <h1 class="h1smaller">Add New Admin/User</h1>
+        <p>Enter the required details below:</p>
     </div>
 
-    <!-- Reset Password Form -->
-    <form action="/open-account" method="POST">
-        <!-- Hidden field for token -->
-        <label for="Name">Name</label>
-        <input type="name" id="name" name="name" required>
-        <label for="email">Email</label>
-        <input type="email" id="email" name="email" required>
+    <form action="/add-user" method="POST">
+        <!-- Admin First Name -->
+        <label for="fname">First Name (Admin)</label>
+        <input type="text" id="fname" name="fname" value="John" required placeholder="Enter first name">
+
+        <!-- User Last Name -->
+        <label for="lname">Last Name (User)</label>
+        <input type="text" id="lname" name="lname" value="Doe" required placeholder="Enter last name">
+
+        <!-- User Email -->
         <label for="email">Email</label>
-        <input type="email" id="email" name="email" required>
-        <label for="newPassword">New Password</label>
-        <input type="password" id="newPassword" name="newPassword" required placeholder="Enter your new password">
+        <input type="email" id="email" name="email" value="johndoe@example.com" required placeholder="Enter email address">
+
+        <!-- User Phone Number -->
+        <label for="phonenum">Phone Number</label>
+        <input type="text" id="phonenum" name="phonenum" value="123-456-7890" required placeholder="Enter phone number">
+
+        <!-- User Phone Carrier -->
+        <label for="carrier">Phone Carrier</label>
+        <input type="text" id="carrier" name="carrier" value="Verizon" required placeholder="Enter phone carrier">
 
-        <label for="confirmPassword">Confirm New Password</label>
-        <input type="password" id="confirmPassword" name="confirmPassword" required placeholder="Re-enter your new password">
+        <!-- User Password -->
+        <label for="pw">Password</label>
+        <input type="password" id="pw" name="pw" value="SecurePassword123!" required placeholder="Enter password">
 
+        <!-- Date of Birth -->
+        <label for="dob">Date of Birth</label>
+        <input type="date" id="dob" name="dob" required placeholder="Select date of birth">

```

Comment:
The date picker kinda sucks but I guess it's the default lol, it's fine for now we can always deal with that later if we want to